docs/features/network-security-controls/admin-network-policy.md

```diff
@@ -1,303 +1,1351 @@
+## Introduction
 
-Kubernetes AdminNetworkPolicy API reference: https://github.com/kubernetes-sigs/network-policy-api/blob/429a9e6ae89d411f89d5a16aba38a5d920c969ee/apis/v1alpha1/adminnetworkpolicy_types.go
+Being able to enforce airtight application security at the cluster-wide level
+has been a popular ask from cluster administrators. Key admin user stories include:
 
-NOTE: This documentation focuses on OVNK's implementation of ANP and is more for developers than for users.
+* As a cluster administrator, I want to enforce non-overridable admin network
+  policies that have to be adhered to by all user tenants in the cluster, thus
+  securing my cluster’s network traffic.
+* As a cluster administrator, I want to implement specific network restrictions
+  across multiple tenants (a tenant here refers to one or more namespaces),
+  thus facilitating network multi-tenancy.
+* As a cluster administrator, I want to enforce the tenant isolation model
+  (a tenant cannot receive traffic from any other tenant) as the cluster’s default
+  network security standard, thus delegating the responsibility of explicitly allowing
+  traffic from other tenants to the tenant owner using NetworkPolicies.
 
-NetworkPolicy API was designed mainly for namespace owners or application developers. They are thus namespace scoped. NetworkPolicy API is not suitable for administrators/network-administrators/security-operators of the cluster because of two main reasons:
+AdminNetworkPolicy has been designed precisely to solve these problems. It provides
+a comprehensive cluster-wide network security solution for cluster administrators using
+two cluster-scoped CRDs:
 
-* They are not cluster-scoped and hence its hard to define a network of policies that spawns across namespaces
-* They cannot be created before the namespace is created (kapi server expects the namespace to be created first); thus they cannot satisfy the requirements where admins may want policies in the cluster to be in place before the workloads are even created.
-* The design of NetworkPolicy API is implicit which means the deny is already set in place when we create a policy and then we are expected to do an allowList of rules in the policy. Network administrators prefer to have the power of defining what exactly to deny and allow instead of this implicit model.
+* AdminNetworkPolicy API and
+* BaselineAdminNetworkPolicy API
 
-Sample API:
+which are shipped by the [sig-network-policy-api](https://network-policy-api.sigs.k8s.io/)
+working group as out-of-tree Kubernetes APIs. Both these APIs can be used to define
+cluster-wide admin policies in order to secure your cluster’s network traffic. These APIs
+are currently in the v1alpha1 version and subject to change based on upstream developments
+before becoming stable v1.
 
+## Motivation
+
+NetworkPolicies in Kubernetes were created for developers to be able to secure their
```

Comment:
should we add a link to the NetworkPolicies pointing to the API to clarify which object we mean here?

Reply:
yep done.